apps/wallet-mobile/src/features/Discover/useCases/SelectDappFromList/DAppListItem/DAppListItem.tsx

```diff
@@ -42,7 +45,16 @@ export const DAppListItem = ({dApp, connected, onPress}: Props) => {
   const {track} = useMetrics()
 
   const HEIGHT_SCREEN = useWindowDimensions().height
-  const heightDialogByHeightScreen = (HEIGHT_SCREEN * 40) / 100
+
+  // For devices like iPhone SE wuth logical pixel height < 700, we need a bottom sheet with more height
+  const heightDialogByHeightScreen = dApp.isSingleAddress
+    ? HEIGHT_SCREEN < 700
+      ? (HEIGHT_SCREEN * 70) / 100
+      : (HEIGHT_SCREEN * 50) / 100
+    : HEIGHT_SCREEN < 700
+    ? (HEIGHT_SCREEN * 50) / 100
+    : (HEIGHT_SCREEN * 40) / 100
```

Comment:
looks solid, yet I think for readability looks simpler
```suggestion
  const heightDialogByHeightScreen = HEIGHT_SCREEN < 700
    ? (HEIGHT_SCREEN * (dApp.isSingleAddress ? 70 : 50)) / 100
    : (HEIGHT_SCREEN * (dApp.isSingleAddress ? 50 : 40)) / 100
```